make_BAF_igv.py

```diff
@@ -0,0 +1,97 @@
+#! /usr/bin/env python3
+import sys
+import argparse
+import vcf
+
+
+def calc_baf(args):
+    if args.outputfile:
+        output_file = open(args.outputfile, 'w')
+
+    open_mode = "r"
+    if args.compressed:
+        open_mode = "rb"
+
+    with open(args.inputfile, open_mode) as vcf_input_file:
+        vcf_reader = vcf.Reader(vcf_input_file)
+        if 'fileformat' not in vcf_reader.metadata:  # Check if true VCF file
+            sys.exit(
+                "Input file {} is not a correct VCF file. "
+                "Field \"fileformat\" was not detected in VCF".format(
+                    args.inputfile
+                )
+            )
+        if len(vcf_reader.samples) > 1:  # Check is VCF is not a multisample VCF
+            sys.exit("Single sample VCF support only. Input file {} is a multisample VCF".format(args.inputfile))
+
+        sampleid = vcf_reader.samples[0]
+
+        header = (
+            "#track type=igv name=BAF_{} color=0,100,0 altColor==0,100,0 "
+            "graphType=points windowingFunction=none maxHeightPixels=100 viewLimits=0,100"
+            "\nchromosome\tstart\tend\tlocus\tbaf".format(
+                sampleid
+            )
+        )
+        if args.outputfile:
+            output_file.write("{}\n".format(header))
+        else:
+            print(header)
+
+        for record in vcf_reader:  # Parse all variant positions, and determine BAF if possible.
+            chrom = record.CHROM
+            end = record.POS
+            start = end - 1
+            locus = "{}:{}-{}".format(chrom, start, end)
+
+            if 'DP' in record.genotype(sampleid).data._asdict():
```

Comment:
for this if statement I would make the code simpler. for example in a line block. something like this:


```
ad = record.genotype(sampleid)['AD'] if dp >= args.mindepth and variant_call  [record.genotype(sampleid)['AD']]: elif dp >= args.mindepth and not variant_call and  'AD' in record.genotype(sampleid).data._asdict() and record.num_unknown == 0 else continue
baf = (ad[1] / dp) * 100 if dp >= args.mindepth and variant_call  ((dp - ad[0]) / (dp)) * 100:  elif dp >= args.mindepth and not variant_call and  'AD' in record.genotype(sampleid).data._asdict() and record.num_unknown == 0 else continue
```